lib/iris/fileformats/netcdf.py

```diff
@@ -84,7 +84,7 @@
              'coordinates', '_FillValue', 'formula_terms', 'grid_mapping',
              'leap_month', 'leap_year', 'long_name', 'missing_value',
              'month_lengths', 'scale_factor', 'standard_error_multiplier',
-             'standard_name', 'units', 'valid_max', 'valid_min', 'valid_range']
```

Comment:
Apologies for butting in, but I think @cpelley has a strong case here: iris' current behaviour is actively preventing the end user from doing the right thing.  His suggested change allows the user to do the right thing now, with the option for iris to automatically infer the right thing at a later date.  It seems to me to be a reasonable interim solution: immediate benefit, with a complete solution to follow.